products/llm_analytics/frontend/LLMAnalyticsScene.tsx

```diff
@@ -338,3 +336,11 @@ export function LLMAnalyticsScene(): JSX.Element {
         </BindLogic>
     )
 }
+
+function truncateValue(value: unknown): string {
+    if (value == null) {
+        return '-'
+    }
+    const stringValue = value.toString()
+    return stringValue.slice(0, 4) + '...' + stringValue.slice(-4)
+}
```

Comment:
For short strings, there's a bug (it was previously there), so let's fix that too:

```suggestion
function truncateValue(value: unknown): string {
    if (value === null || value === undefined) {
        return '-'
    }
    
    const stringValue = String(value)
    
    if (stringValue.length <= 12) {
        return stringValue
    }
    
    return stringValue.slice(0, 4) + '...' + stringValue.slice(-4)
}
```